Source/WebCore/style/RuleSetBuilder.cpp

```diff
@@ -516,6 +519,10 @@ void RuleSetBuilder::addMutatingRulesToResolver()
             if (m_ruleSet)
                 m_ruleSet->setViewTransitionRule(*styleRuleViewTransition);
         }
+        if (auto* styleRulePositionTry = dynamicDowncast<StyleRulePositionTry>(rule.get())) {
+            // FIXME: do something here...
+            // FIXME: newline to make style checker happy for the time being
+        }
```

Comment:
I would just remove this for now, and re-add it when you have an implementation of the evaluation ready.

Reply:
I'd remove this for now.